### PR TITLE
DHFPROD-8356: tooltip flapping

### PR DESCRIPTION
**Pull request by MSBrignoni**
### Description
Tooltip Flapping

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Run UI tests
  

- ##### Reviewer:

- [x] Reviewed Tests
- [N/A] Added Release Notes

### CODE REVIEW

**Review by aniamy — APPROVED** [review]

File: marklogic-data-hub-central/ui/src/components/load/load-card.tsx [hunk 1]
```diff
@@ -465,7 +465,7 @@ const LoadCard: React.FC<Props> = (props) => {
 
                   <div className={styles.cardNonLink} data-testid={`${elem.name}-toExistingFlow`}>
                                     Add step to an existing flow
-                    {selectVisible ? <HCTooltip id="missing-permission-tooltip" text={"Load: "+SecurityTooltips.missingPermission} placement={"bottom"}><div className={styles.cardLinkSelect}><div className={styles.cardLinkSelect}>
+                    {selectVisible ? <HCTooltip id="missing-permission-tooltip" show={!props.canWriteFlow} text={"Load: "+SecurityTooltips.missingPermission} placement={"bottom"}><div className={styles.cardLinkSelect}><div className={styles.cardLinkSelect}>
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/src/components/load/load-card.tsx:468]: Should we have a unit tests that covers this scenario?

**Review by Sanjeevani19 — COMMENTED** [review]
@MSBrignoni Please add a test scenario to handle this bugfix

**Comment by AkshayBajajML** [comment]
Converting it to draft meanwhile.

File: marklogic-data-hub-central/ui/src/components/load/load-card.tsx (hunk 1 above)
Comment (MSBrignoni) [reply marklogic-data-hub-central/ui/src/components/load/load-card.tsx]: Yes, was added a new e2e for this scenario, after different probes, is more stable than UI tests.

**Comment by MSBrignoni** [comment]
> @MSBrignoni Please add a test scenario to handle this bugfix

Done, an e2e was added.

**Review by briantang — CHANGES_REQUESTED** [review]
@MSBrignoni Sometimes, there seems to be an extra ghost rendering of the tooltip in the top left of the screen: https://watch.screencastify.com/v/0JG7DYwk7jiKeQQYpn4u

**Comment by briantang** [comment]
@csilva-ml Is it preferable to create new shorter spec files like in this PR? Or does it make more sense to add this into one of the existing specs that test the same area (e.g. defaultIngestionCardView.spec).

**Comment by MSBrignoni** [comment]
> @MSBrignoni Sometimes, there seems to be an extra ghost rendering of the tooltip in the top left of the screen: https://watch.screencastify.com/v/0JG7DYwk7jiKeQQYpn4u

Thanks @briantang , not was happening in FF so i had to test this in chrome, i added a change in the show property of the tooltip.

**Review by briantang — APPROVED** [review]
Looks good!

**Comment by csilva-ml** [comment]
> @csilva-ml Is it preferable to create new shorter spec files like in this PR? Or does it make more sense to add this into one of the existing specs that test the same area (e.g. defaultIngestionCardView.spec).

@briantang yes, our current spec files are too long, dependent and mostly flaky. I recommend creating smaller new files using our new standards. We need to come up with a plan on to how to organize them better in subfolders but for now this is great work @MSBrignoni

**Review by aniamy — APPROVED** [review]

**Review by Sanjeevani19 — APPROVED** [review]

**Pull request merged**